assets/css/admin-bar-rtl.css

```diff
@@ -0,0 +1,1182 @@
+	-moz-osx-font-smoothing: grayscale;
+	background-image: none !important;
+	margin-left: 6px;
+}
+
+#wpadminbar .ab-icon:before,
+#wpadminbar .ab-item:before,
+#wpadminbar #adminbarsearch:before {
+	color: #a0a5aa;
+	color: rgba(240, 245, 250, 0.6);
+}
+
+#wpadminbar .ab-icon:before,
+#wpadminbar .ab-item:before,
+#wpadminbar #adminbarsearch:before {
+	position: relative;
+	transition: all .1s ease-in-out;
+}
+
+#wpadminbar .ab-label {
+	display: inline-block;
+	height: 32px;
+}
+
+#wpadminbar .ab-submenu .ab-item {
+	color: #b4b9be;
+	color: rgba(240, 245, 250, 0.7);
+}
+
+#wpadminbar .quicklinks .menupop ul li a,
+#wpadminbar .quicklinks .menupop ul li a strong,
+#wpadminbar .quicklinks .menupop:focus-within ul li a,
+#wpadminbar.nojs .quicklinks .menupop:hover ul li a {
+	color: #b4b9be;
+	color: rgba(240, 245, 250, 0.7);
+}
+
+#wpadminbar .quicklinks .menupop ul li a:hover,
+#wpadminbar .quicklinks .menupop ul li a:focus,
+#wpadminbar .quicklinks .menupop ul li a:hover strong,
+#wpadminbar .quicklinks .menupop ul li a:focus strong,
+#wpadminbar .quicklinks .ab-sub-wrapper .menupop:focus-within > a,
+#wpadminbar .quicklinks .menupop:focus-within ul li a:hover,
+#wpadminbar .quicklinks .menupop:focus-within ul li a:focus,
+#wpadminbar .quicklinks .menupop:focus-within ul li div[tabindex]:hover,
+#wpadminbar .quicklinks .menupop:focus-within ul li div[tabindex]:focus,
+#wpadminbar.nojs .quicklinks .menupop:hover ul li a:hover,
+#wpadminbar.nojs .quicklinks .menupop:hover ul li a:focus,
+#wpadminbar li:hover .ab-icon:before,
+#wpadminbar li:hover .ab-item:before,
+#wpadminbar li a:focus .ab-icon:before,
+#wpadminbar li .ab-item:focus:before,
+#wpadminbar li .ab-item:focus .ab-icon:before,
+#wpadminbar li:focus-within .ab-icon:before,
+#wpadminbar li:focus-within .ab-item:before,
+#wpadminbar li:hover #adminbarsearch:before,
+#wpadminbar li #adminbarsearch.adminbar-focused:before {
+	color: #00b9eb;
+}
+
+#wpadminbar.mobile .quicklinks .ab-icon:before,
+#wpadminbar.mobile .quicklinks .ab-item:before {
+	color: #b4b9be;
+}
+
+#wpadminbar.mobile .quicklinks :focus-within .ab-icon:before,
+#wpadminbar.mobile .quicklinks :focus-within .ab-item:before {
+	color: #00b9eb;
+}
+
+#wpadminbar .menupop .menupop > .ab-item .wp-admin-bar-arrow:before,
+#wpadminbar .ab-top-secondary .menupop .menupop > .ab-item:before {
+	position: absolute;
+	font: normal 17px/1 dashicons;
+	speak: none;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+#wpadminbar .menupop .menupop > .ab-item {
+	display: block;
+	padding-left: 2em;
+}
+
+#wpadminbar .menupop .menupop > .ab-item .wp-admin-bar-arrow:before {
+	top: 1px;
+	left: 10px;
+	padding: 4px 0;
+	content: "\f141";
+	color: inherit;
+}
+
+#wpadminbar .ab-top-secondary .menupop .menupop > .ab-item {
+	padding-right: 2em;
+	padding-left: 1em;
+}
+
+#wpadminbar .ab-top-secondary .menupop .menupop > .ab-item .wp-admin-bar-arrow:before {
+	top: 1px;
+	right: 6px;
+	content: "\f139";
```

Comment:
This should be `content: "\f141";` in RTL (opposite of the one in the default CSS)

Reply:
(FWIW I ran `admin-bar.css` through `RTLCSS` locally, and this was changed)